sstables/kl/writer.cc

```diff
@@ -663,7 +663,7 @@ void sstable_writer_k_l::consume_end_of_stream()
         features.disable(sstable_feature::NonCompoundRangeTombstones);
     }
     run_identifier identifier{_run_identifier};
-    _sst.write_scylla_metadata(_pc, _shard, std::move(features), std::move(identifier));
+    _sst.write_scylla_metadata(_pc, _shard, std::move(features), std::move(identifier), false);
```

Comment:
Why K/L sstables always have the new metadata set to false? Out of laziness or is there a reason?

Reply:
Laziness and I only really needed the metadata for >= md

Reply:
I'd like to understand this as well. if we set it to false by default, the metadata will be made available, and if it's set incorrectly, that could lead to a wrong decision down the road.

Reply:
I think it's safe because the metadata either says there's no partition tombstone or we don't know. Like a bloom filter.

Reply:
false means we know nothing partition tombstones are there or not.

Reply:
so perhaps we shouldn't even bother writing it for older versions

Reply:
we do have the feature bit set in the scylla file however so that should be fine backward compatibility wise